src/acceptance-test/java/org/zalando/nakadi/webservice/timelines/TimelineConsumptionTest.java

```diff
@@ -18,14 +19,12 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 import org.zalando.nakadi.domain.EventType;
-import org.zalando.nakadi.webservice.BaseAT;
-import static com.jayway.restassured.RestAssured.given;
 import static org.zalando.nakadi.webservice.utils.NakadiTestUtils.createEventType;
 import static org.zalando.nakadi.webservice.utils.NakadiTestUtils.createTimeline;
 import static org.zalando.nakadi.webservice.utils.NakadiTestUtils.deleteTimeline;
 import static org.zalando.nakadi.webservice.utils.NakadiTestUtils.publishEvent;
 
-public class TimelineConsumptionTest extends BaseAT {
+public class TimelineConsumptionTest {
```

Comment:
@lmontrieux Cause it is not needed. BaseAT is a rudiment of db initialization. I do not need it, and actually want to remove BaseAT at all, cause acceptance tests should perform tests or running nakadi instance, without hacks with initialization and record drops